src/index.js

```diff
@@ -405,6 +441,7 @@ export default class RNPickerSelect extends PureComponent {
                     value={selectedItem.inputLabel ? selectedItem.inputLabel : selectedItem.label}
                     ref={this.setInputRef}
                     editable={false}
+                    onLayout={this.onTextInputLayout}
```

Comment:
I think this needs to be updated to `scrollToInput`, right?

Reply:
No, this was just an unused callback i forgot to remove. Thanks for pointing that out! :)